services/whisperer/index.js

```diff
@@ -0,0 +1,25 @@
+const { default: Whisperer } = require('@coding-blocks/whisperer');
+const config = require('../../config/config');
+
+class WhispererService {
+  constructor(connectionParams) {
+    this._whisperer = new Whisperer({
+      ...connectionParams
+    })
+  }
+
+  initialize() {
+    return this._whisperer.init()
+  }
+
+  emit(topic, msg) {
+    this._whisperer.emit(topic, msg)
+  }
+}
+
+module.exports = new WhispererService({
```

Comment:
why create a wrapper around it?
```
module.exports = new Whisperer({ ... }})
```

Reply:
As of now, we don't need a wrapper but in the future if we need some abstractions from the API itself I thought it'll be useful